### PR TITLE
nautilus: qa/tests: advanced nautilus initial version to 14.2.20

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by yuriw** [comment]
https://pulpito.ceph.com/teuthology-2021-05-07_16:24:00-upgrade:nautilus-p2p:nautilus-p2p-parallel-nautilus-distro-basic-smithi
https://pulpito.ceph.com/teuthology-2021-05-07_16:25:47-upgrade:nautilus-p2p:nautilus-p2p-stress-split-nautilus-distro-basic-smithi

File: qa/suites/upgrade/nautilus-p2p/nautilus-p2p-stress-split/1-ceph-install/nautilus.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 meta:
-- desc: install ceph/nautilus v14.2.2
+- desc: install ceph/nautilus v14.2.19
 tasks:
 - install:
-    tag: v14.2.2
+    tag: v14.2.19
```
Comment (jdurgin) [inline qa/suites/upgrade/nautilus-p2p/nautilus-p2p-stress-split/1-ceph-install/nautilus.yaml:5]: why not .20 here?

Comment (neha-ojha) [reply qa/suites/upgrade/nautilus-p2p/nautilus-p2p-stress-split/1-ceph-install/nautilus.yaml:5]: I don't remember any related issues, perhaps, you are confusing it with https://tracker.ceph.com/issues/50595

**Review by jdurgin — APPROVED** [review]

**Review by neha-ojha — APPROVED** [review]
nit: you can squash the commits into one

**Pull request merged**